migrations/2022-01-04-034553_add_hidden_column/up.sql

```diff
@@ -0,0 +1,13 @@
+alter table community add column hidden boolean default false;
+
+
+create table mod_hide_community
+(
+    id serial primary key,
+    community_id int references community on update cascade on delete cascade not null,
+    person_id int references person on update cascade on delete cascade not null,
+    when_ timestamp not null default now(),
+    reason text not null,
```

Comment:
Looks good, except don't make this required, IE remove the not null to be uniform with the others mod tables. If we decide we want to make those all required, we'll do that all at the same time, at a later date.